packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -46,3 +79,65 @@ export function sanitizeUser(user: User): PublicUser {
 	const { password, resetPasswordToken, createdAt, updatedAt, ...sanitizedUser } = user;
 	return sanitizedUser;
 }
+
+async function getCredentialIdByName(name: string): Promise<string | number | null> {
+	const credential = await Db.collections.Credentials!.findOne({
+		where: {
+			name,
+		},
+	});
+	if (credential) {
+		return credential.id;
+	}
+	return null;
+}
```

Comment:
this doesn't work anymore, since names are not unique!

Reply:
Sorry, I removed this feature but this code was left. It was not being used anymore.